src/commands/tokens/create.ts

```diff
@@ -28,15 +31,37 @@ export default class TokenCreateCommand extends Command {
   @hasArg
   description: string;
 
+  @help("The type of token principal authentication: [" + allPrincipalTypes.join(", ") + "]")
+  @shortName("t")
+  @longName("type")
+  @hasArg
+  @defaultValue("user")
+  public principalType: string;
```

Comment:
Any reason to make this `public`?

Reply:
This formatting matched other CLI command arguments (I was looking at the distribute ones for reference), but no I didn't actually realize that the previous parameters for these commands weren't marked as public. Would you prefer without `public`?